resources-generator/src/main/kotlin/dev/icerock/gradle/generator/FontsGenerator.kt

```diff
@@ -92,7 +93,7 @@ abstract class FontsGenerator(
         fontStyleFiles
             .forEach { (styleName, file) ->
                 val styleProperty = PropertySpec
-                    .builder(styleName.decapitalize(), resourceClassName)
+                    .builder(styleName.replaceFirstChar { it.lowercase(Locale.ROOT) }, resourceClassName)
```

Comment:
please add extension function `decapitalize` and move this code to function. hard to read this code

Reply:
fixed